pkg/agent/openflow/pipeline.go

```diff
@@ -1032,7 +991,7 @@ func (f *featurePodConnectivity) flowsToTrace(dataplaneTag uint8,
 			if ovsMetersAreSupported {
 				fb = fb.Action().Meter(PacketInMeterIDTF)
 			}
-			fb = fb.Action().SendToController(uint8(PacketInReasonTF))
+			fb = fb.Action().SendToController(uint8(PacketInCategoryTF), PacketInNoMultiOperation, false)
```

Comment:
Is it possible to merge the userData in featurs, then use type `[]byte` for the parameter in SendToController? Then the `FlowBuilder` could not understand the contents of the callers.

Reply:
I have changed to use `[]byte` for `SendTo Controller`. 
As for merging it into features, it's a little difficult. Because a feature could use multiple packet-in categories, like `featureNetworkPolicy` it could use `PacketInCategoryNP` and `PacketInCategoryTF`. Also a packet-in category could be used by multiple features, like `PacketInCategoryTF` is used by `featureNetworkPolicy` and `featurePodConnectivity`.

Reply:
I didn't mean to use "feature" to express the openflow features we implement in path pkg/agent/openflow, I meant to use a single catagoray for a separate use case, like traceflow, NP with logging, NP with FQDN, multicast, Service with reject, etc. So here I am asking if we have some special cases must share one catagory. If not, we could just let the caller to provide the final userdata which would be filled in the OF message.

Reply:
Actually, we do. For the NP category, one `packetIn` may require multiple operations. For example, it could need logging and reject at the same time.